src/api/spec/mailers/event_mailer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe EventMailer do
+  # Needed for X-OBS-URL
+  before { Configuration.any_instance.stubs(:obs_url).returns('https://build.example.com') }
+
+  context 'comment mail' do
+    let!(:receiver) { create(:confirmed_user) }
+    let!(:subscription) { create(:event_subscription_comment_for_project, user: receiver) }
+    let!(:comment) { create(:comment_project, body: "Hey @#{receiver.login} how are things?") }
```

Comment:
After using 3 `let!` for setting up the subscriptions and comment... what is better? using a before block with those lines?, or this way? Maybe the before block is more readable...